docs/release_notes.rst

```diff
@@ -1,6 +1,11 @@
 Release Notes
 =============
 
+v4.3.2
+----------
+
+* Fix discrepancy between runtime and type-checker's perspective of Index and derived types
```

Comment:
Will this be edited later to include all changes between 4.3.1 and 4.3.2?

Reply:
Missed that one!